src/test/java/org/testng/internal/paramhandler/FakeTestNGMethod.java

```diff
@@ -0,0 +1,379 @@
+package org.testng.internal.paramhandler;
+
+import org.testng.IClass;
+import org.testng.IRetryAnalyzer;
+import org.testng.ITestClass;
+import org.testng.ITestNGMethod;
+import org.testng.internal.ConstructorOrMethod;
+import org.testng.internal.MethodInstanceTest;
+import org.testng.internal.reflect.ReflectionHelper;
+import org.testng.xml.XmlClass;
+import org.testng.xml.XmlTest;
+
+import java.lang.reflect.Method;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Callable;
+
+public class FakeTestNGMethod implements ITestNGMethod {
+    private ConstructorOrMethod consMethod;
```

Comment:
final

Reply:
Cannot do this, because `consMethod` is being initialized within a condition. So IntelliJ complains saying it may not have been initialized.

Reply:
you can use a temp var for that specific case :)

Reply:
fixed.